lisa/sut_orchestrator/azure/common.py

```diff
@@ -352,12 +362,12 @@ def shared_gallery(self) -> Optional[SharedImageGallerySchema]:
                 r"[/]+", self.shared_gallery_raw.strip().lower()
             )
             if len(shared_gallery_strings) == 5:
-                shared_gallery = SharedImageGallerySchema(*shared_gallery_strings)
+                shared_gallery = SharedImageGallerySchema(True, *shared_gallery_strings)
```

Comment:
It's an optional field, doesn't need to provide value. Create a method like `create` to initialize a schema, so it would be easier.

Reply:
Do you mean to add a "create" function in the parent class?
I have modified it as the parameters are set to value separately. Does that look fine?